src/quantuminspire/api.py

```diff
@@ -665,6 +680,7 @@ def get_asset(self, asset_id: int) -> Dict[str, Any]:
                 | contentType (str)         | The description of the content e.g. 'application/qasm' or
                 |                           | 'text/plain'.
                 | content (str)             | The content itself. For example a cQASM program when linked to a job.
+                | measurement_mask (int)    | A mask for the measured bits in the cQASM program.
```

Comment:
Maybe specify here that this is a read-only parameter that gets calculated when the asset is assigned to a job.

Reply:
All parameters are read only in SDK. I will add the extra remark when it is calculated